Sources/Impl/PinLayoutImpl+Coordinates.swift

```diff
@@ -361,26 +361,26 @@ extension PinLayoutImpl {
         }
     }
     
-    fileprivate func computeCoordinates(_ point: CGPoint, _ layoutSuperview: PView, _ referenceSuperview: PView) -> CGPoint {
+    fileprivate func computeCoordinates(_ point: CGPoint, _ layoutSuperview: View, _ referenceSuperview: View) -> CGPoint {
         if layoutSuperview == referenceSuperview {
             return point   // same superview => no coordinates conversion required.
-        } else if referenceSuperview == layoutSuperview.superview {
+        } else if referenceSuperview == layoutSuperview.superview as? View {
             let layoutSuperviewRect = layoutSuperview.getRect(keepTransform: keepTransform)
             return CGPoint(x: point.x - layoutSuperviewRect.origin.x,
                            y: point.y - layoutSuperviewRect.origin.y)
         // TOOD: Handle all cases. computeCoordinates should compute coordinates using only untransformed
         //       coordinates, but UIView.convert(...) below use transformed coordinates!
         //       Currently we only support 1 and 2 levels.
         } else {
-            return referenceSuperview.convert(point, to: layoutSuperview)
+            return referenceSuperview.convert(point, toView: layoutSuperview as! View.View)
```

Comment:
It could be `layoutSuperview as? View.View`. 
Also, `View.View` is a little bit weird 🙂, in my branch I was using `TLayoutable` instead of `View` for the generic type name. What do you think?